System/Metrics.hs

```diff
@@ -332,9 +332,6 @@ createDistribution name store = do
 -- function.
 
 #if MIN_VERSION_base(4,10,0)
--- | Convert nanoseconds to milliseconds.
-nsToMs :: Int64 -> Int64
```

Comment:
Can this break anything?

Reply:
I don't think so, as I removed all usages of this function.

Reply:
I was rather thinking of clients breaking due to ms -> ns change. We should mention it in the changelog.

Reply:
Yes, the fields and their meanings definitely change completely.